modules/rhdh-plugins-reference/proc-setting-up-openshift-pipelines-to-view-detailed-scan-reports.adoc

```diff
@@ -0,0 +1,142 @@
+        - name: report
+          image: 'quay.io/lrangine/crda-maven:11.0'
+          script: |
+              #!/bin/sh
+              cat $(workspaces.reports.path)/image-scan
+----
+<1> Task name. For example, acs-image-scan, acs-image-check, acs-deployment-check
+<2> The supported result format
+<3> Type of scan tasks. For example, `roxctl-image-scan`, `roxctl-image-check`, and `roxctl-deployment-check`. You can create separate tasks in pipeline for these scan tasks.
+<4> The name of the container to get the POD logs.
+
+.. For EC, update your existing scan task with annotations in the following format:
++
+[source,yaml]
+----
+...
+metadata:
+ name: enterprise-contract-task <1>
+ annotations:
+    task.results.format: application/json <2>
+    task.results.type: ec <3>
+    task.output.location: logs
+    task.results.container: step-report-json <4>
+spec: …
+      steps:
+        - name: report-json
+          image: quay.io/enterprise-contract/ec-cli:snapshot@sha256:33be4031a3316a46db3559a4d8566bc22f9d4d491d262d699614f32f35b45b67 <5>
+          command: [cat]
+          args:
+            - "$(params.HOMEDIR)/report-json.json"
+----
+<1> Name of the task, such as enterprise-contract-task.
+<2> Format supported for the result.
+<3> Type of scan tool, here ec.
+<4> Container name for retrieving POD logs.
+<5> Image URL for the scanning tool.
+
+. Rerun the affected OpenShift Pipeline.
+
+
+.Verification
+
+. Log in to Red Hat Developer Hub.
+. Select *Catalog* and open an appropriate project for which you updated the pipeline in Openshift console.
+. Select the *CI* tab > *Actions* column > *View output* icon and review the detailed vulnerabilities reports in the software components.
+
+
+== Interpreting vulnerability and compliance scan reports
+
+The detailed reports generated by Advanced Cluster Security (ACS) and Enterprise Contract (EC) tasks are instrumental in providing security insights crucial for maintaining a robust security posture. 
+
+.The ACS and EC tasks in a pipeline
+image::rhdh-plugins-reference/acs-ec.png[]
+
+Here's how to interpret these reports:
+
+*ACS scan insights*
+
+* *Vulnerability Breakdown:* ACS scans categorize detected vulnerabilities by severity (Critical, Important, Moderate, Low), status (fixable, non-fixable), and offer a summary of the scan results. This categorization includes the total number of vulnerabilities and components analyzed, alongside specific Common Vulnerabilities and Exposures (CVEs) identified.
+
+* *Details Provided:* For each identified vulnerability, the report includes:
+
+** *CVE ID:* A unique identifier for the vulnerability.
+** *Severity:* The level of threat posed by the vulnerability.
+** *Component:* The software component affected by the vulnerability.
+** *Component Version:* The version of the affected component.
+** *Remediation Suggestions:* Recommendations for addressing the vulnerability, including the version in which the vulnerability is fixed, if applicable.
+
```

Comment:
Can we have ACS scan report screenshot here ? 

Refer this screenshot - https://github.com/janus-idp/backstage-plugins/blob/main/plugins/tekton/images/tekton-plugin-acs-output.png